charts/modules/apps/crossplane/values.yaml

```diff
@@ -38,10 +38,10 @@ global:
     name: crossplane-provider-config-aws
   ## @param global.awsProviderVersion Crossplane official AWS provider version: https://marketplace.upbound.io/providers/upbound/provider-aws/
   ##
-  awsProviderVersion: v0.47.1
+  awsProviderVersion: v0.41.0
```

Comment:
Upgrade to 0.47.1 is okay. Its works well. I think we could keep this upgrade to avoid CRD version downgrades.

Reply:
See release notes for 0.43 - https://github.com/upbound/provider-aws/releases/tag/v0.43.1. Are you sure that mixing 1.13 and 1.14 together is safe?

Reply:
0.47.x aws provider on 1.13.x crossplane worked well for us but it does makes sense to keep default provider version to 0.41.0 to make sure crossplane runtime version in provider release matches with crossplane version.

For our deployment, we should override provider version to 0.47.1 because CRDs in the cluster are already upgraded. If we notice any issues, we can go for the downgrade. Hoping that we don't have to do that.